app/helpers/importHelpers.php

```diff
@@ -1370,14 +1370,14 @@ function caPhpExcelGetDateCellContent($po_sheet, $pn_row_num, $pm_col, $pn_offse
 	# ---------------------------------------------------------------------
 	/**
 	 * Get raw cell from Excel sheet for given column and row
-	 * @param PHPExcel_Worksheet $po_sheet The work sheet
+	 * @param \PhpOffice\PhpSpreadsheet\Worksheet\Worksheet $po_sheet The work sheet
 	 * @param int $pn_row_num row number (zero indexed)
 	 * @param string|int $pm_col either column number (zero indexed) or column letter ('A', 'BC')
-	 * @return PHPExcel_Cell|null the cell, if a value exists
+	 * @return \PhpOffice\PhpSpreadsheet\Cell\Cell|null the cell, if a value exists
 	 */
 	function caPhpExcelGetRawCell($po_sheet, $pn_row_num, $pm_col) {
 		if(!is_numeric($pm_col)) {
-			$pm_col = PHPExcel_Cell::columnIndexFromString($pm_col)-1;
+			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col)-1;
```

Comment:
In PHPSpreadsheet coordinates are 1 based, so the code should read

```suggestion
			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col);
```